src/mappings/control/events/orgCreated.ts

```diff
@@ -76,10 +76,7 @@ async function handleOrgCreatedEvent(ctx: Context, block: Block, event: Event, n
     org.url = metadata?.url ?? ''
     org.location = metadata?.location ?? ''
     org.tags = metadata?.tags ?? []
-
-    // TODO: pottential duplicates, there is no uniqueness check implemented
-    // TODO: if the "name" was updated on IPFS, we have no callback to update the slug
-    org.slug = slugify(org.name)
+    org.slug = slugify(org.name + '-' + org.id)
```

Comment:
In this case, we need to either:
 - create an additional slug field in createOrg extrinsic and check uniqueness on-chain. This might be quite slow as every new name the user checks requires chain calls.
 - create a default slug in squid (e.g. name+id) and provide users a way to change it. In this case, the slug would be stored in squid DB, not on the chain.

I tend to the latter approach as it would benefit user experience.
WDYT @2075 @vayesy ?